src/logic.js

```diff
@@ -23,7 +23,7 @@ define([
             "hintItext",
         ],
         NO_SELF_REFERENCES = _.without(XPATH_REFERENCES, 'constraintAttr'),
-        CASE_REF_ID = "#case/";
+        EXTERNAL_REF = "#";
```

Comment:
Yeah. I think `#` is good enough for now, but i can see someone getting confused in the future. I think there's a comment in here about not splitting the case references as it gets rid of important inofrmation. This could be fixed if that is ever fixed